### PR TITLE
Found new predefined controller coefficients for qcp-su

### PR DESCRIPTION
**Pull request by fdamken**
The old ones just didn't work anymore for whatever reason. These work for 100Hz, 250Hz, and 500Hz in simulation.

### CODE REVIEW

File: Pyrado/pyrado/policies/special/environment_specific.py [hunk 1]
```diff
@@ -166,8 +166,8 @@ def __init__(self, env_spec: EnvSpec, long: bool = False, use_cuda: bool = False
             self._log_K_pd_init = to.log(
                 to.tensor([41.0, 200.0, 55.0, 16.0])
             )  # former: [+41.8, 173.4, +46.1, 16.2], [34.1, 118.0, 43.4, 18.1]
-            self._log_k_e_init = to.log(to.tensor(17.0))  # former: 24.5, 36.5, 19.5
-            self._log_k_p_init = to.log(to.tensor(8.0))  # former: 4.0, 8.5, 2.25
+            self._log_k_e_init = to.log(to.tensor(40.588629412818705))  # former: 17.0  # former: 24.5, 36.5, 19.5
+            self._log_k_p_init = to.log(to.tensor(3.3650026417213748))  # former: 8.0  # former: 4.0, 8.5, 2.25
```
Comment (famura) [inline Pyrado/pyrado/policies/special/environment_specific.py:170]: please remove the second '# former'

File: Pyrado/scripts/hyperparam_optimization/hopt_sb_qcp-su.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        simple_dynamics=True,
+        wild_init=False,
+    )
+    policy = QCartPoleSwingUpAndBalanceCtrl(env.spec)
+    # policy._log_K_pd.data = to.tensor([
+    #     trial.suggest_uniform("log_K_pd_1", 0.0, 300.0),
+    #     trial.suggest_uniform("log_K_pd_2", 0.0, 300.0),
+    #     trial.suggest_uniform("log_K_pd_3", 0.0, 300.0),
+    #     trial.suggest_uniform("log_K_pd_4", 0.0, 300.0),
+    # ])
```
Comment (famura) [inline Pyrado/scripts/hyperparam_optimization/hopt_sb_qcp-su.py:65]: where are these not subject to optimization?

File: Pyrado/scripts/hyperparam_optimization/hopt_sb_qcp-su.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+from pyrado.environments.pysim.quanser_cartpole import QCartPoleSwingUpSim
+from pyrado.logger.experiment import save_dicts_to_yaml, setup_experiment
+from pyrado.policies.special.environment_specific import QCartPoleSwingUpAndBalanceCtrl
+from pyrado.sampling.rollout import rollout
+from pyrado.utils.argparser import get_argparser
+from pyrado.utils.input_output import print_cbt
+
+
+def train_and_eval(trial: optuna.Trial):
+    dt = 0.004
```
Comment (famura) [inline Pyrado/scripts/hyperparam_optimization/hopt_sb_qcp-su.py:49]: add doc like for the other hparam optimization scripts

**Review by famura — CHANGES_REQUESTED** [review]
Most importantly: have you tested these new values for k_e and k_p on the real cart-pole?

File: Pyrado/scripts/hyperparam_optimization/hopt_sb_qcp-su.py (hunk 2 above)
Comment (fdamken) [reply Pyrado/scripts/hyperparam_optimization/hopt_sb_qcp-su.py:65]: Stabilization worked well in sim and these are only used during stabilization. So I removed them from the optimization to reduce the search space.

**Comment by fdamken** [comment]
> Most importantly: have you tested these new values for k_e and k_p on the real cart-pole?

No

File: Pyrado/scripts/hyperparam_optimization/hopt_sb_qcp-su.py (hunk 2 above)
Comment (famura) [reply Pyrado/scripts/hyperparam_optimization/hopt_sb_qcp-su.py:72]: fine. however, I guess th ebayesian opitmization can handle 4 more params. just saying, when you do the effort anyways...

**Comment by fdamken** [comment]
They don't work on the real cartpole.

**Comment by famura** [comment]
Thank you for testing it. Did you actually also test the existing param config on that day?

**Pull request closed without merging**